tests/integration/store_and_load_test.py

```diff
@@ -1,34 +1,202 @@
-import mwclient
-
 import osw.model.entity as model
+from osw.auth import CredentialManager
 from osw.core import OSW
+from osw.utils.wiki import get_full_title
 from osw.wiki_tools import SearchParam
 from osw.wtsite import WtSite
 
 # run with: tox -e test -- --wiki_domain domain --wiki_username user --wiki_password pass
 
 
 def test_store_and_load(wiki_domain, wiki_username, wiki_password):
```

Comment:
Beside this integration test we should add a unit test for apply_override_policy()

Reply:
This implies that apply_overwrite_policy() should be accessible independently of store_entity() - correct?

Reply:
Correct =)

Reply:
implemented in https://github.com/OpenSemanticLab/osw-python/pull/61/commits/b073c50acf0fb6e114707f3c96b6704c9e159cbd
test included with https://github.com/OpenSemanticLab/osw-python/pull/61/commits/22bac86933129484f8aa7553b52dd6ce0dd4eb53